src/components/panel/panel.js

```diff
@@ -304,7 +310,7 @@ angular
  *
  * @param {string} newClass class to be added.
  * @param {boolean} toElement Whether or not to add the class to the panel
- *    element instead of the container.
+ * element instead of the container.
```

Comment:
This should be indented as it's a continuation from the param line above. If this were not in a param and instead a description, this would be fine.